### PR TITLE
fix: tune test_replicaof_reject_on_load parameters

### PR DESCRIPTION
**Pull request by romange**
Reduce the snapshot size by 20% and increase the timeout to avoid failures due to slow loads.
Fixes https://github.com/dragonflydb/dragonfly/actions/runs/10982136864/job/30490032552

### CODE REVIEW

File: tests/dragonfly/replication_test.py [hunk 1]
```diff
@@ -2014,7 +2014,7 @@ async def test_replicaof_reject_on_load(df_factory, df_seeder_factory):
     df_factory.start_all([master, replica])
 
     c_replica = replica.client()
-    await c_replica.execute_command(f"DEBUG POPULATE 10000000")
+    await c_replica.execute_command(f"DEBUG POPULATE 8000000")
```
Comment (adiholden) [inline tests/dragonfly/replication_test.py:2017]: this is a very slow test, takes lots of time. Borys should be working on fixing it (not by tuning the params)
hope to see a PR soon

Comment (BorysTheDev) [reply tests/dragonfly/replication_test.py:2017]: I still don't have an idea how to improve it without adding some debug commands additional info

Comment (romange) [reply tests/dragonfly/replication_test.py:2017]: I think this test is not worth spending time on. Lets merge and move on.

Comment (adiholden) [reply tests/dragonfly/replication_test.py:2017]: please mark this as a slow test and you can also add shutdown no save when stopping the replica to reduce the test time

**Review by kostasrim — APPROVED** [review]

File: tests/dragonfly/replication_test.py (hunk 1 above)
Comment (romange) [reply tests/dragonfly/replication_test.py:2017]: sorry, it was marked as automerge

**Pull request merged**